Jenkinsfile

```diff
@@ -147,18 +126,22 @@ pipeline{
 				}
 			}
 		}
-		// This stage emails the contents of the GraphQA_Summary_vXX.csv that is generated by graph-qa for review. 
+		// This stage emails the contents of the GraphQA_Summary_vXX.csv that is generated by graph-qa for review, as well as the GraphQA_Summary_vXX-1 (previous release). 
 		stage('Post: Email graph-qa output'){
 			steps{
 				script{
+					def prevGraphQAFileName = "GraphQA_Summary_v${previousRelease}.csv"
+					def s3PathPrevGraphQA = "${env.S3_RELEASE_DIRECTORY_URL}/${previousRelease}/orthoinference/reports/${prevGraphQAFileName}.gz"
+					sh "aws s3 cp ${s3PathPrevGraphQA} ."
```

Comment:
A line of whitespace between the variable definitions and the commands might be good (not mandatory - just a thought).